### DISCUSSION TITLE
example只有basic版本吗，有没有那个工程运行起来以后功能和官网示例是一样的

### DISCUSSION THREAD

**Discussion by gtrgeg** [alibaba/designable#390] [Q&A]
如题，basic版本太简单了。是否有和https://designable-antd.formilyjs.org/功能一致的版本的源代码提供给我们。
题外话：https://designable.netlify.app/ 这个演示地址打不开，控制台报错
![屏幕截图 2023-11-07 155132](https://github.com/alibaba/designable/assets/47472797/448aee05-58ec-4949-8a49-dec728493a5c)

**Comment by gtrgeg** [comment]
已经找到入口了

**Reply by goodshileidev** [reply]
入口在哪里？我也没找到

**Comment by zaoren** [comment]
是的。因为antd最新的已经升级到5.0了
<script src="https://unpkg.com/antd/dist/antd-with-locales.min.js"></script> cdn 应该替换成  4.x版本的
